BP/localization/english/culture/cultures_l_english.yml

```diff
@@ -0,0 +1,392 @@
+﻿l_english: 
+ # Baltic Group
```

Comment:
Since this file adds content not in vanilla it should be called BP_cultures_l_english

Reply:
Aight.

Reply:
Fixed